integrationTests/state/stateTrie/stateTrie_test.go

```diff
@@ -1755,6 +1717,16 @@ func collectSnapshotAndCheckpointHashes(
 	}
 }
 
+// the rounds for the blocks on epoch e that are confirmed by a block that gets finalized in the epoch e+1
+//should be skipped from pruning
+func isRoundSkippedForPruning(nonce uint64, roundsPerEpoch uint64) bool {
+	skip:= nonce%roundsPerEpoch > roundsPerEpoch-2
+	if skip{
+		fmt.Println(fmt.Sprintf("skipping nonce %d", nonce))
```

Comment:
from linter: `S1038: should use fmt.Printf instead of fmt.Println(fmt.Sprintf(...)) (but don't forget the newline) (gosimple)`
or use log.Info
Also, unused function

Reply:
removed the unused func